used-tasks-tags

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# Determine all tasks container tags that are used in our projects.
+# Used for cleaning up obsolete tasks tags.
+
+import asyncio
+import re
+import sys
+
+from lib import testmap
+from lib.aio.base import SubjectSpecification
+from lib.aio.github import GitHub
+from lib.directories import xdg_config_home
+
+
+async def main() -> None:
+    used_tags = set()
+
+    # avoid rate limiting with token
+    try:
+        with open(xdg_config_home("cockpit-dev/github-token")) as f:
```

Comment:
In GitHub workflows we'd rather use `os.env.get('GITHUB_TOKEN')` I assume?

Reply:
I actually wrote that first, but then checked and saw that there is no precedence for that. Our workflows usually do this in bots:
```
.github/workflows/image-trigger.yml:        run: echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
.github/workflows/naughty-prune.yml:        run: echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
.github/workflows/naughty-prune.yml:          echo ${{ github.token }} >> ~/.config/cockpit-dev/github-token
```

or cockpit:
```
.github/workflows/nightly.yml:          echo "${{ github.token }}" >> ~/.config/cockpit-dev/github-token
.github/workflows/tasks-container-update.yml:          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
```

(yes, we are not very consistent wrt. the old and new path here)